tests/testthat/test-pipe-consistency-linter.R

```diff
@@ -0,0 +1,148 @@
+test_that("pipe_consistency skips allowed usage", {
+  skip_if_not_r_version("4.1.0")
+  expect_lint("1:3 %>% mean() %>% as.character()", NULL, pipe_consistency_linter())
+  expect_lint("1:3 |> mean() |> as.character()", NULL, pipe_consistency_linter())
+  # With no pipes
+  expect_lint("x <- 1:5", NULL, pipe_consistency_linter())
+  # Across multiple lines
+  expect_lint(
+    trim_some("
+      1:3 %>%
+        mean() %>%
+        as.character()
+    "),
+    NULL,
+    pipe_consistency_linter()
+  )
+})
+
+test_that("pipe_consistency lints inconsistent usage", {
+  skip_if_not_r_version("4.1.0")
+  expected_msg <- rex::rex("Found 1 instances of %>% and 1 instances of |>. Stick to one pipe operator.")
```

Comment:
We've imported `rex()` into the namespace now, so you no longer need to ns-qualify.

Reply:
NB: This is a test script, we don't attach {rex}:

https://github.com/r-lib/lintr/blob/dec0f34c9a2eae59824463bcd165445db3f4e87a/tests/testthat.R#L1-L8